frontends/web/src/routes/settings/advanced-settings.tsx

```diff
@@ -82,6 +87,9 @@ export const AdvancedSettings = ({ deviceIDs, hasAccounts }: TPagePropsWithSetti
               >
                 <EnableCustomFeesToggleSetting frontendConfig={frontendConfig} onChangeConfig={setConfig} />
                 <EnableCoinControlSetting frontendConfig={frontendConfig} onChangeConfig={setConfig} />
+                { runningInAndroid() && (
```

Comment:
i would move the `runningInAndroid()` check inside the `EnableAuthSetting` component so it is self-contained, but no big deal.

Reply:
Done